pyserini/index/faiss.py

```diff
@@ -35,6 +35,7 @@
     parser.add_argument('--pq-m', type=int, default=192, required=False)
     parser.add_argument('--pq-nbits', type=int, default=8, required=False)
     parser.add_argument('--threads', type=int, default=12, required=False)
+    parser.add_argument('--flatmetric', type=string, default="inner", required=False)
```

Comment:
why `flatmetric` and not just `metric`?

Reply:
both are specific for the flat index, but, likely, you should pick a metric and apply it to either the flat index or one of the other index builds.

Reply:
So we should rename to `--metric` then? @MXueguang @crystina-z thoughts?